src/Microsoft.ML.FastTree/FastTreeClassification.cs

```diff
@@ -130,14 +129,14 @@ public sealed partial class FastTreeBinaryClassificationTrainer :
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
             string weightColumn = null,
-            int numLeaves = Defaults.NumLeaves,
-            int numTrees = Defaults.NumTrees,
-            int minDatapointsInLeaves = Defaults.MinDocumentsInLeaves,
-            double learningRate = Defaults.LearningRates)
+            int numLeaves = Defaults.NumberOfLeaves,
```

Comment:
>numLeaves [](start = 16, length = 9)

same suggestion here to use the full names for the arguments.

Reply:
Sure.

---
In reply to: [261461521](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261461521) [](ancestors = 261461521)